minefield_game/init.cpp

```diff
@@ -1,13 +1,18 @@
 #include <iostream>
 #include <vector>
-#include "init_header.h"
-#include "cell_header.h"
-#include "player_header.h"
+#include "init.h"
+#include "cell.h"
+#include "player.h"
 namespace Init
 {
-    bool rangeValidation(int const size) { return size > 24 && size <= 50; }
+    int constexpr minBoardSize = 24;
+    int constexpr maxBoardSize = 50;
+    int constexpr minMines = 3;
+    int constexpr maxMines = 8;

```

Comment:
#nice use of `constexpr`!

#should use a naming convention different than the one for variables on these constants

#should make them `static`